react/features/toolbox/components/web/Toolbox.js

```diff
@@ -264,7 +264,9 @@ const styles = theme => {
             fontSize: 14,
             listStyleType: 'none',
             padding: '8px 0',
-            backgroundColor: theme.palette.ui03
+            backgroundColor: theme.palette.ui03,
+            overflowY: 'auto',
+            height: 'calc(80vh - 64px - 144px - 16px)' // drawer max height - reactions menu height - padding
```

Comment:
nit- move relevant values to consts, and if the drawer and reactions are moved already to js css, use them there as well.

Reply:
The styles for drawer and reactions were not in JS, but I moved them :)